docker/arch.Dockerfile

```
@@ -0,0 +1,45 @@
+FROM archlinux:latest
+
+RUN pacman -Syu --noconfirm \
+  && pacman -S --noconfirm php
+
+ARG RELAY=v0.8.1
+
+# Install Relay dependencies
+RUN pacman -S --noconfirm \
+  base-devel \
+  # hiredis \
+  libck \
+  lz4
+
+# Install hiredis-ssl
+RUN curl -L https://github.com/redis/hiredis/archive/refs/tags/v1.2.0.tar.gz | tar -xzC /tmp \
+  && PREFIX=/usr USE_SSL=1 make -C /tmp/hiredis-1.2.0 install
+
+# Install php-pear
+RUN curl -L https://aur.archlinux.org/cgit/aur.git/snapshot/php-pear.tar.gz | sudo -u nobody tar -xzC /tmp \
+  && pushd /tmp/php-pear \
+  && sudo -u nobody makepkg --skippgpcheck \
+  && pacman -U --noconfirm php-pear-1:1.10.23-2-any.pkg.tar.zst \
+  && popd
+
+# Relay requires the `msgpack` extension
+RUN pecl install msgpack \
+  && echo "extension = msgpack.so" > $(php-config --ini-dir)/msgpack.ini
+
+# Relay requires the `igbinary` extension
+RUN pecl install igbinary \
+  && echo "extension = igbinary.so" > $(php-config --ini-dir)/igbinary.ini
+
+# Download Relay
+RUN ARCH=$(uname -m | sed 's/_/-/') \
+  PHP=$(php -r 'echo substr(PHP_VERSION, 0, 3);') \
```

How about a robust solution?
```suggestion
  PHP=$(php -r 'printf("%d.%d", PHP_MAJOR_VERSION, PHP_MINORR_VERSION);') \
```

---

I don't mind, but it should be done in all dockerfiles